gfx/UltimateCataclysm/pngs_centered_64x64/f_electrical_conduit/f_electrical_conduit.json

```
@@ -0,0 +1,34 @@
+{
+  "id": "f_electrical_conduit",
+  "multitile": true,
+  "fg": "f_electrical_conduit_unconnected",
+  "bg": "",
+  "additional_tiles": [
+	{
```

```suggestion
	  {
```
It will look better ( At least in **Notepad++** ) if you made the **opening curly brace** in the same **column/space** as the **closing curly brace**